ludwig/models/modules/measure_modules.py

```diff
@@ -158,3 +158,25 @@ def r2(targets, predictions, output_feature_name):
     r2 = tf.subtract(1., res_ss / tot_ss,
                      name='r2_{}'.format(output_feature_name))
     return r2
+
+
+def bbox_iou(targets, predictions, output_feature_name):
+    x11, y11, x12, y12 = tf.split(targets, 4, axis=1)
```

Comment:
Not sure why are you transposing few lines from here, can you add a comment on what shapes do you expcet the inputs to have?

Reply:
I removed the transposed variables.

Reply:
Was looking into literature, it seems like it is more common to provide bounding boxes as: x, y h, w where x and y are the coordinates of the center, h w are height and width respectively.
Can you please doublecheck a couple implementations like faster rcnn and yolo to make sure we are doing the most common thing?

Reply:
Yes, I will have look over the week

Reply:
Moreover, it seems to me that the bounding box output feature has been placed in the input features. Am I reading that correctly?